inbox/auth/utils.py

```diff
@@ -37,42 +37,31 @@ def auth_requires_app_password(exc: IMAPClient.Error) -> bool:
     )
 
 
-# IMAP doesn't really have error semantics, so we have to match the error
-# message against a list of known response strings to determine whether we
-# couldn't log in because the credentials are invalid, or because of some
```

Comment:
I think losing `or because of some temporary server error` is a mistake, because it makes the purpose of this mechanism less clear.

Reply:
I don't know, it just makes it more obvious what we're aiming for, when you add an "as opposed to" (btw, "or because" wasn't the best phrasing, and should probably have been "as opposed to").

I don't think it's that big of a deal, but I don't think it hurts to keep it. Feel free to ignore me here.

Reply:
I think that's unnecessary. If I'm testing a string "to determine something", it is automatically implied that it is "as opposed to it not being that something".